### PR TITLE
[Magiclysm] Spell description changes

### PR DESCRIPTION
**Pull request by Milopetilo**
#### Summary
Mods "[Magiclysm] Change some spell description to feel more natural/be more accurate"

#### Purpose of change
I noticed that in #71236 the part of "lashing tentacles" was changed to the shoulders, but the description of the spell was unaffected. Then I noticed some of the descriptions for other spells were sometimes inaccurate or had weird sentences, so I tried to spruce them up a bit.

#### Describe the solution
Change the description of the spell to match its effects. Change some weirdly built sentences to (hopefully) easier to read ones.

#### Describe alternatives you've considered
Change it so the tentacles come out of your torso instead of shoulders.
Keep the descriptions the same.

#### Testing
Went in-game, noticed the changes happened in the spell descriptions.

### CODE REVIEW

File: tools/spell_checker/dictionary.txt [hunk 1]
```diff
@@ -849,7 +849,8 @@ diverges
 divil
 dizzied
 dodecahedron
-doesnt
+doesn
+doesn't
```
Comment (GuardianDll) [inline tools/spell_checker/dictionary.txt:853]: dictionary doesn't handle apostrophes at all, it will read it as words `doesn`, `t`

Comment (Milopetilo) [reply tools/spell_checker/dictionary.txt:853]: Should I add `doesn` and `isn` then or just ignore the dictionary and remove my changes there?

Comment (GuardianDll) [reply tools/spell_checker/dictionary.txt:853]: remove and ignore it, someone need to fix it in another, proper way

File: tools/spell_checker/dictionary.txt [hunk 2]
```diff
@@ -849,7 +849,6 @@ diverges
 divil
 dizzied
 dodecahedron
-doesnt
```
Comment (Maleclypse) [inline tools/spell_checker/dictionary.txt:852]: I don't think you should be removing this as it didn't come from any of the changed lines in this PR.

**Pull request merged**